values.yaml

```diff
@@ -802,6 +802,22 @@ connectInject:
   image: null # image for consul-k8s that contains the injector
   default: false # true will inject by default, otherwise requires annotation
 
+  # Health checks enables propagation of k8s pod health status to Consul service instances.
+  # This has the effect of marking a Consul service instance passing/critical in the case
+  # that the connect-injected pod has passed/failed health checks.
+  # Setting this to false will cause the health check status to not be synced.
+  # Toggling this on/off is not recommended as it may leave stale Consul health checks that
+  # will eventually expire and be marked critical which would require manual cleanup or
```

Comment:
I'm not sure it's worth calling it out here since it implies that you won't restart your consul clients for 11yrs (or so), which I think is pretty unlikely.